### PR TITLE
Adapt test suite for OCP 4.9

### PR DESCRIPTION
**Pull request by mkralik3**
- AMQ Streams was updated to 1.8 (in order to use crd v1 api)
- Jaeger repo fix
- XTF and Kubernetes client were updated
- Workaround for missing CRD class (`NoClassDefFoundError`) ( it was moved to v1beta1 package) since Syndesis use old Kubernetes client. It is needed until the Kubernetes client will be updated in Syndesis too ( https://github.com/syndesisio/syndesis/pull/9777 )
- Changes in order to still support OpenShift3

### CODE REVIEW

File: endpoints/pom.xml [hunk 1]
```diff
@@ -81,5 +81,11 @@
 			<groupId>org.assertj</groupId>
 			<artifactId>assertj-core</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>io.fabric8</groupId>
+			<artifactId>kubernetes-model-apiextensions</artifactId>
+			<version>4.13.3</version>
```
Comment (avano) [inline endpoints/pom.xml]: `${openshift.client.version}`

**Comment by syndesisqe** [comment]
Checkstyle verification failed with:
```
endpoints/src/main/java/io/fabric8/kubernetes/api/model/apiextensions/
  CustomResourceDefinition.java:1: File does not end with a newline. [NewlineAtEndOfFile]
endpoints/src/main/java/io/fabric8/kubernetes/api/model/apiextensions/
  CustomResourceDefinition.java:9:126: '{' is not followed by whitespace. [WhitespaceAround]
endpoints/src/main/java/io/fabric8/kubernetes/api/model/apiextensions/
  CustomResourceDefinition.java:9:127: '}' is not preceded with whitespace. [WhitespaceAround]
```

File: utilities/src/main/java/io/syndesis/qe/resource/impl/CamelK.java [hunk 2]
```diff
@@ -120,12 +120,13 @@ private void extractArchive() {
...
     private CustomResourceDefinitionContext getCamelKCRD() {
-        CustomResourceDefinition crd = OpenShiftUtils.getInstance().customResourceDefinitions().withName("integrations.camel.apache.org").get();
+        CustomResourceDefinition crd =
+            OpenShiftUtils.getInstance().apiextensions().v1().customResourceDefinitions().withName("integrations.camel.apache.org").get();
         CustomResourceDefinitionContext.Builder builder = new CustomResourceDefinitionContext.Builder()
             .withGroup(crd.getSpec().getGroup())
             .withPlural(crd.getSpec().getNames().getPlural())
             .withScope(crd.getSpec().getScope())
-            .withVersion(crd.getSpec().getVersion());
+            .withVersion(crd.getSpec().getVersions().stream().filter(v -> v.getServed().equals(true)).findFirst().get().getName());
```
Comment (avano) [inline utilities/src/main/java/io/syndesis/qe/resource/impl/CamelK.java]: `.filter(v -> v.getServed())` or `.filter(CustomResourceDefinitionVersion::getServed)`

**Review by avano — DISMISSED** [review]

**Comment by mkralik3** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Review by avano — APPROVED** [review]

**Pull request merged**